targets/ath79-generic

```diff
@@ -76,3 +76,9 @@ device('ocedo-raccoon', 'ocedo_raccoon', {
 device('tp-link-archer-c6-v2', 'tplink_archer-c6-v2', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
+
+-- Ubiquity
+
+device('ubiquiti-nanostation-ac', 'ubnt_nanostation-ac', {
+	packages = ATH10K_PACKAGES_QCA9888,
```

Comment:
Wrong packages: OpenWRT uses ath10k-firmware-qca988x-ct firmware, so you need to use ATH10K_PACKAGES_QCA9880.

Reply:
Good catch, my bad.